float32[6] joints_values
---
bool success

